### DISCUSSION TITLE
how to enable|use preset

### DISCUSSION THREAD

**Discussion by bilak** [renovatebot/renovate#17286] [Request Help] [answered]
If there are defined [presets](https://docs.renovatebot.com/presets-group/#groupspringboot) am I able to use that preset somehow? Something like `"enableGroups": ["spring boot"]`?

**Comment by viceice** [comment, marked as answer]
- https://docs.renovatebot.com/key-concepts/presets/
- https://docs.renovatebot.com/configuration-options/#extends

**Reply by bilak** [reply]
so this is the way to go?
```
"extends": [
    "group:springBoot"
  ]
```
